src/main.js

```diff
@@ -186,17 +186,26 @@ function takeMeBack() {
   savedPostersView.classList.add('hidden');
 }
 
+function pushValues() {
+  images.push(image.src);
+  titles.push(title.innerText);
+  quotes.push(quote.innerText);
+}
+
 function showUserPoster() {
-  // event.preventDefault(event);
-  var userImage = imageInput.value;
-  var userTitle = titleInput.value;
-  var userQuote = quoteInput.value;
-  image.src = userImage;
-  title.innerText = userTitle;
-  quote.innerText = userQuote;
-  console.log(userImage, userTitle, userQuote);
-    // figure out how to import/require poster.js, so that we can start using our class!
-  currentPoster = new Poster(userImage, userTitle, userQuote);
+  event.preventDefault(event);
+  // var userImage = imageInput.value;
+  // var userTitle = titleInput.value;
+  // var userQuote = quoteInput.value;
+  // image.src = userImage;
+  // title.innerText = userTitle;
+  // quote.innerText = userQuote;
+  image.src = imageInput.value;
+  title.innerText = titleInput.value;
+  quote.innerText = quoteInput.value;
+  // console.log(userImage, userTitle, userQuote);
+  currentPoster = new Poster(imageInput.value, titleInput.value, quoteInput.value);
```

Comment:
I see you've added takeMeBack() to currentPoster as a 'new class' instantiation parameter - will simply invoking showUserPoster() do the trick and send us back to the main page, then?

Reply:
on line 208? If this is what you're referring to, the function takeMeBack() is there to add and remove the hidden classes from those HTML sections. I'm not sure we discussed those during our call. Do you have questions about how this one works?